app/models/vm_or_template/operations/configuration.rb

```diff
@@ -103,4 +103,28 @@ def spec_reconfigure(spec)
     raise _("VM has no EMS, unable to apply reconfigure spec") unless ext_management_system
     run_command_via_parent(:vm_reconfigure, :spec => spec)
   end
+
```

Comment:
I think this code should go to ManageIQ::Providers::CloudManager::Vm (floating IPs work only for Cloud Provider, right?)